talon/quotations.py

```diff
@@ -40,9 +40,9 @@
             'Am',
             # Norwegian
             u'På',
+            # Vietnamese
```

Comment:
Looks like the comment was moved here by mistake: the Vietnamese pattern is actually below - where the comment used to be.